test/ch/imedias/rsccfx/model/RsccTest.java

```diff
@@ -21,92 +20,126 @@
   private static final String KEY_SERVER_HTTP_PORT = "800";
 
   Rscc model;
-  ProcessExecutor mockProcessExecutor;
+  SystemCommander mockProcessExecutor;
 
   /**
    * Initializes test fixture before each test.
    */
-  @Ignore
   @Before
   public void setUp() throws Exception {
-    mockProcessExecutor = mock(ProcessExecutor.class);
-    model = new Rscc(new SystemCommander());
+    mockProcessExecutor = mock(SystemCommander.class);
+    model = new Rscc(mockProcessExecutor);
     model.keyServerSetup(KEY_SERVER_IP, KEY_SERVER_HTTP_PORT);
-    when(mockProcessExecutor.getOutput()).thenReturn("OUTPUT>" + KEY);
+    when(mockProcessExecutor.executeTerminalCommand(
+        argThat(string -> string.contains("start_x11vnc.sh")))).thenReturn(KEY);
+  }
+
+  /**
+   * Test for Constructor {@link Rscc#Rscc(SystemCommander)}.
+   */
+  @Test
+  public void testRsccConstructorIllegalArguments() {
+    try {
+      new Rscc(null);
+      fail("Test has to fail");
+    } catch (IllegalArgumentException e) {
+      e.getMessage();
```

Comment:
since this doesn't do anything but make the tests run slower (the output is not being used in any way) I would suggest to replace this with a comment like: "// expected behavior"